egs/librispeech/ASR/pruned_transducer_stateless3/scaling_converter.py

```diff
@@ -164,6 +170,38 @@ def scaled_embedding_to_embedding(
     return embedding
 
 
+def scaled_lstm_to_lstm(scaled_lstm: ScaledLSTM):
```

Comment:
Please add type hint for the return value.

Reply:
Ok. Thanks.